### DISCUSSION TITLE
article/video-clip

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#28] [Announcements]
# article/video-clip

我喜欢剪辑视频，用图像、声音这样更具象、更生动的形式，向人传达一些情感。尤其是当用自己喜欢的音乐做BGM嵌入视频，根据歌词的内容填充视频画面、用画面的转场卡点音乐的节奏，莫名的就会觉得舒服。

https://tangly1024.com/article/video-clip

**Comment by txs** [comment]
我之前都用 FCPX 剪，但 FCPX 會中轉 ProRes 導致佔很大的空間，現在我改用剪映直接剪，直接輸出 H265，省空間也省時間，唯一的缺點是一次只能輸出一個影片，輸出的時候無法編輯下一個專案。

**Reply by tangly1024** [reply]
剪映使用起来不像FCPX那么顺手，导出视频的话我习惯用Compressor，功能很强大😁
剪映最吸引我还是他的特效和素材，其它方面FCPX还是更胜一筹~
